devops/poolservice/poolservice.profile

```diff
@@ -0,0 +1 @@
+export ConnectionString=""
```

Comment:
Is it set as empty here just to remember it's a mandatory config?

Reply:
if we ever automate deployment that's where we'd fill in the AppConfig string.  For now, that's the only part that's a manual step.